db/models/aggregate_functions.py

```diff
@@ -153,14 +153,14 @@ def submit_application(application_id):
 def update_form(application_id, form_name, question_json):
     try:
         form_sql_row = FormsMethods.get_form(application_id, form_name)
-        form_sql_row.json = question_json
-        db.session.commit()
-        update_statuses(application_id, form_name)
-        if form_sql_row != question_json:
+        if form_sql_row.json[0]["fields"] != question_json[0]["fields"]:
             current_app.logger.info(
                 f"Application updated for application_id: '{application_id}."
             )
             ApplicationsMethods.application_edited(application_id)
```

Comment:
Should this be moved below line 162, as lines 161 and 162 update the question JSON and statuses in the db. I think this move would make your logging statement more accurate as the update will have happened? 👀

Reply:
nice spot! have changed this now to log after running the query :D